messages/agent.preview.md

```diff
@@ -28,23 +28,24 @@ Before you use this command, you must complete these steps:
 
 3. Copy the consumer key from your connected app as described in the "Obtain Credentials" section here: https://developer.salesforce.com/docs/einstein/genai/guide/agent-api-get-started.html#obtain-credentials.
 
-4. Set the "SFDX_AUTH_SCOPES" environment variable to "refresh_token sfap_api chatbot_api web api". This step ensures that you get the specific OAuth scopes required by this command.
-
-5. Using the username of the user you specified as the "Run As" user above, authorize your org using the web server flow, as described in this document: https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm.
+4. Link the connected app to your authenticated user with the required API scopes using the web server flow:
+```
```

Comment:
I'm pretty sure that these backticks show up literally in `--help` so I would remove them for now. 

BTW, I plan to remove almost all this info from the --help and simply point to the Agentforce Dev Guide, with a note to say "see [this page](https://developer.salesforce.com/docs/einstein/genai/guide/agent-dx-preview.html) for required configuration before you can run this command" or something like that.   Feel free to do that as part of this PR!  It might be cleaner.  And then I will update that Dev Guide page appropriately.